tests/acceptance/features/apiGraph/createGroup.feature

```diff
@@ -21,6 +21,8 @@ Feature: create group
     | comma,grp1      |
     | Finance (NP)    |
     | slash\Middle    |
+    | !               |
+    | =               |
```

Comment:
I added this cases because `=` fails because of https://github.com/owncloud/ocis/issues/5970 and we'll know when it is fixed